zenoh-flow/src/types.rs

```diff
@@ -190,6 +262,11 @@ impl ZFState for EmptyState {
     }
 }
 
+/// The default output rules are a commodity function provided to
+/// users that do not need output rules in their operators.
+/// It converts all the incoming data to the right node output.
+///
+/// It never returns an error.
```

Comment:
I would replace with:

Commodity function for users that do not need output rules in their operators.
The data is simply converted to the expected ```[`NodeOutput`](`NodeOutput`)``` type.